starlingcaptureapi/actions.py

```diff
@@ -22,18 +23,102 @@
 class Actions:
     """Actions for processing assets."""
 
-    def archive(self, asset_meta_path: str):
+    def archive(self, organization_id, collection_id, zip_path, asset_exts, aes_key):
```

Comment:
I think this should accept the same params as other actions in @anaulin's PR:

```
asset_fullpath, org_config, collection_id):
```

Then the `archive` action contains this:

```
                "encryption":
                {
                  "algo": "aes-256-cbc",
                  "key": "starlinglab-aes-256"
                }
```

Read from this, and handle accordingly. I think this involves:

1. Fetching `starlinglab-aes-256` as `aes_key`
2. If it fails, generate and store one for the collection named `starlinglab-aes-256` in a known key directory (e.g. `KEY_STORE="~/.starling-integrity/key-store/"` in config.json)

If we want one-key-per-collection, we'd set up the config to have `"key": "collection_id_aes256.key"`).